### ISSUE TITLE
Repetitive Styling

### ISSUE THREAD

**Issue by josephgupkim** [alysonj614/prj-rev-bwfs-dasmoto#2]
If this were a project with more styling needs, I'm worried that your code base would have been unnecessarily large due to the repetitive styling that I noticed.
You could have accomplished this more efficiently by using an "h2" selector and assigning it the repetitively used attributes.

https://github.com/alysonj614/prj-rev-bwfs-dasmoto/blob/master/Dasmotos/resources/style.css#L13-L32

**Comment by alysonj614** [comment]
Thanks so much for your comments.  I looked over the h2 sections and realized how I could have written it more efficiently.  Just wondering if the below code is better? 

h2 {
  font-size: 32px;
  font-weight: bold;
  color: white;
}

.brushes h2 {
  background-color: mediumspringgreen;
}

.frames h2 {
  background-color: lightcoral;
}

.paints h2 {
  background-color: skyblue;
}

**Comment by josephgupkim** [comment]
Yes! This would've definitely been the more efficient approach. Great job! :)

**Comment by alysonj614** [comment]
awesome.  thanks so much :)
